android/src/main/kotlin/vn/casperpas/sound_stream/SoundStreamPlugin.kt

```diff
@@ -317,7 +317,16 @@ public class SoundStreamPlugin : FlutterPlugin,
                 .setUsage(AudioAttributes.USAGE_MEDIA)
                 .setFlags(AudioAttributes.FLAG_AUDIBILITY_ENFORCED)
                 .build()
-        mAudioTrack = AudioTrack(audioAttributes, mPlayerFormat, mPlayerBufferSize, AudioTrack.MODE_STREAM, AudioManager.AUDIO_SESSION_ID_GENERATE)
+//        mAudioTrack = AudioTrack(audioAttributes, mPlayerFormat, mPlayerBufferSize, AudioTrack.MODE_STREAM, AudioManager.AUDIO_SESSION_ID_GENERATE)
+
+        mAudioTrack = AudioTrack(AudioManager.STREAM_MUSIC,
```

Comment:
This AudioTrack construction is deprecated.

Reply:
Ohh. Did you found any other alternative for the same?